trove/render/simple_tsv.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+import csv
+import functools
+import dataclasses
+import typing
+
+from trove.vocab import mediatypes
+from trove.vocab import osfmap
+from trove.vocab.namespaces import TROVE
+from ._simple_trovesearch import SimpleTrovesearchRenderer
+from ._rendering import StreamableRendering
+
+
+Jsonpath = typing.Iterable[str]  # path of json keys
+
+_MULTIVALUE_DELIMITER = ' ; '  # possible improvement: smarter in-value delimiting?
+_VALUE_KEY_PREFERENCE = ('@value', '@id', 'name', 'prefLabel', 'label')
+
+
+class TrovesearchSimpleTsvRenderer(SimpleTrovesearchRenderer):
+    MEDIATYPE = mediatypes.TAB_SEPARATED_VALUES
+    INDEXCARD_DERIVER_IRI = TROVE['derive/osfmap_json']
+    CSV_DIALECT: type[csv.Dialect] = csv.excel_tab
```

Comment:
Nit: Given this, I'd expect the inheritance between CSV/TSV Renderers to be swapped, but it's not important

Reply:
yeahh i elevated tsv on purpose to emphasize tabs as a preferred delimiter over commas, treating csv as back-compat, but then the standard python import for either is `csv` and naming got confused from there -- agree it would make more sense the other way